testing/src/Environment.php

```diff
@@ -66,10 +66,11 @@ public function startTemporalTestServer(int $commandTimeout = 10): void
         }
     }
 
-    public function startRoadRunner(string $rrCommand = null, int $commandTimeout = 10): void
+    public function startRoadRunner(string $rrCommand = null, int $commandTimeout = 10, array $envs = []): void
     {
         $this->roadRunnerProcess = new Process(
-            $rrCommand ? explode(' ', $rrCommand) : [$this->systemInfo->rrExecutable, 'serve']
+            command: $rrCommand ? explode(' ', $rrCommand) : [$this->systemInfo->rrExecutable, 'serve'],
+            env: $envs
```

Comment:
> \* \@param array|null $env The environment variables or null to use the same environment as the current PHP process
> *\Symfony\Component\Process\Process::__construct()*

Are you sure the default value should be an empty array?
I think it will change the current behavior.

Reply:
It not change the behaviour - https://github.com/symfony/process/blob/6.2/Process.php#L160